docs/ios/README.md

```diff
@@ -0,0 +1,31 @@
+# iOS distribution
+
+The source code can be found in [RNNativePlayground](https://github.com/kiwicom/mobile/tree/master/ios/RNNativePlayground)
+
+## Test playground app
+
+### KiwiHotels module
+
+1. To run `KiwiHotels` module with the packager, change [`RNKiwiConstants.m`](https://github.com/kiwicom/mobile/blob/master/ios/RNKiwiMobile/RNKiwiConstants.m) like follows:
+
+```objc
++ (NSURL *)bundleURL {
+  return [NSURL URLWithString:@"http://localhost:8081/app/native.bundle?platform=ios&dev=true"];
+}
+```
+
+2. Then run the packager from the root of this repo:
+
+```bash
+yarn start
+```
+
+3. Finally just open the project in XCode (use `reactNativeApp.xcworkspace`) and choose `RNNativePlayground` scheme.
+
+![](../../.docs/assets/open_playground.gif)
+
+_NOTE_: If you want to test playground with `jsbundle` go directly to step 3.
+
+## Usage in a real app
+
+Paste the `.framework` within real app.
```

Comment:
You can't really `paste` it. You should break this documentation down into multiple steps, including (but not limited to) the following points:
- Link static library with the target where you want to use the app
- Import RNKiwiViewController and instantiate it
- Push it

Good place to mention pre-initialising the bridge by explicitly calling it. That's also where we will mention constants

Reply:
Oh, just noticed you did that in a different page. Then, either link to it from here, or move it here and link from there. Your choice! 🗡